packages/go/stbernard/workspace/golang/build.go

```diff
@@ -33,15 +33,17 @@ import (
 // BuildMainPackages builds all main packages for a list of module paths
 func BuildMainPackages(workRoot string, modPaths []string, env environment.Environment) error {
 	var (
+		err      error
 		errs     []error
 		wg       sync.WaitGroup
 		mu       sync.Mutex
+		version  semver.Version
 		buildDir = filepath.Join(workRoot, "dist") + string(filepath.Separator)
 	)
 
-	version, err := git.ParseLatestVersionFromTags(workRoot, env)
-	if err != nil {
-		return fmt.Errorf("parse latest version from git tags: %w", err)
+	if version, err = git.ParseLatestVersionFromTags(workRoot, env); err != nil {
+		log.Errorf("Failed to parse version from git tags, falling back to environment variable: %v", err)
```

Comment:
I think this will be expected behavior in some cases. I recommend we move this to a Warnf